documentation/development/cd.md

```diff
@@ -0,0 +1,15 @@
+# Continuous Delivery
+
+This describes aspects of Continuous Delivery setup specifically for OpenCraft,
+and is of limited interested for external contributors.
+
+## Cleaning up left-over resources
+
+Test runs (especially failing ones) may ocasionally leave behind resources that,
+over time, clutter the underlying infrastructure.
+
+To avoid this, there is [a `scheduled-cleanup` CircleCI workflow](https://github.com/open-craft/opencraft/blob/master/circle.yml)
+that regularly runs the `cleanup` job.
+
+If necessary (usually when testing changes to it), the job may be run on-demand
+by manually triggering it in CircleCI.
```

Comment:
@kewne, like I mentioned [here](https://github.com/open-craft/opencraft/pull/815#issuecomment-873970787), CircleCI allows re-running jobs without having to write any code/configuration.